tools/benchmark/cmd/put.go

```diff
@@ -44,6 +49,8 @@ func init() {
 	putCmd.Flags().IntVar(&keySize, "key-size", 8, "Key size of put request")
 	putCmd.Flags().IntVar(&valSize, "val-size", 8, "Value size of put request")
 	putCmd.Flags().IntVar(&putTotal, "total", 10000, "Total number of put requests")
+	putCmd.Flags().IntVar(&keySpaceSize, "key-space-size", 0, "Maximum possible keys")
```

Comment:
Can't write to a random key with keyspace=1 here since the keyspace is bounded to [0, n), but there's still randomizing behavior with a large key space size and `--sequential-keys=false`